human_experiments/scripts/stop_recording.py

```diff
@@ -0,0 +1,21 @@
+import socket
+import subprocess, os
```

Comment:
The `subprocess` and `os` modules are not being used, so you can remove this line.